packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -104,10 +114,21 @@ export default class VTable {
             return;
         }
         this.table.style.borderCollapse = 'collapse';
-        this.trs[0].style.backgroundColor = format.bgColorOdd || 'transparent';
-        if (this.trs[1]) {
-            this.trs[1].style.backgroundColor = format.bgColorEven || 'transparent';
-        }
+        this.setBorderColors(format);

```

Comment:
Suggest not change existing applyFormat(), but add a new API, and mark old one as @deprecated.